java/google/registry/model/reporting/HistoryEntry.java

```diff
@@ -63,6 +63,10 @@
     HOST_PENDING_DELETE,
     HOST_UPDATE,
     /**
+     * Resource was created by an escrow file import.
+     */
+    RDE_IMPORT,
```

Comment:
@CydeWeys I can see your point, however I think that if we use the existing history entry types it will cause confusion later. Another current invariant in the system is that the history entries for these resources also contain the xml that was sent over epp in order to create or modify them. In the case of RDE import, the xml that was used during the import is quite different than the epp xml. By creating a new type of history entry we have a clear way of differentiating between resources created by epp vs those created by rde import.

That said, adding it wasn't my idea. @goldfeder you suggested adding this history entry type when we first discussed the design. Any thoughts on this?